build/configs/esp_wrover_kit/tc/defconfig

```diff
@@ -95,7 +95,7 @@ CONFIG_ESP32_I2C0_MODE_MASTER=y
 # CONFIG_ESP32_I2C0_MODE_SLAVE is not set
 CONFIG_ESP32_I2C0_SCLK_PIN=4
 # CONFIG_ESP32_I2C0_SCLK_PULLUP_EN is not set
-CONFIG_ESP32_I2C0_SDA_PIN=5
+CONFIG_ESP32_I2C0_SDA_PIN=2
```

Comment:
Why did you change the default SDA number?

Reply:
Pin IO5 is used in LCD. Thus, I2C SDA is changed to pin IO2 in wrover kit board.